src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/ApprenticeshipResumedEvent.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events
+{
+    public class ApprenticeshipResumedEvent
+    {
+        public long ApprenticeshipId { get; set; }
```

Comment:
I recommend making the event immutable by adding a constructor and removing the setters.

Reply:
See my previous response (in CV-146)